lang/inputs.go

```diff
@@ -318,6 +318,12 @@ func inputCode(s string, fs engine.Fs) (*ParsedInput, error) {
 		}
 	}
 
+	// Check if code is `metadata.yaml`, which is obviously not correct here
+	if s == interfaces.MetadataFilename {
+		return nil, fmt.Errorf("unexpected raw code '%s'. Did you mean "+
```

Comment:
why did you add the `+` instead of just having a single string?
(you can keep this all on oneline, no need to break at 80 for such a thing)

Reply:
I wasn't sure. gofmt strangely doesn't break this line even though it's a lot longer than 80 chars. I've added what I think is a reasonable break. lmk what you think